pandas/core/groupby/generic.py

```diff
@@ -311,8 +311,8 @@ def _aggregate_multiple_funcs(self, arg):
 
             arg = zip(columns, arg)
 
-        results = {}
-        for name, func in arg:
+        results: Mapping[base.OutputKey, Union[Series, DataFrame]] = {}
```

Comment:
```suggestion
        results: Dict[base.OutputKey, Union[Series, DataFrame]] = {}
```

Nice idea but only need mapping for API exposed signatures; internally if its a dict can just type as such